### PR TITLE
Adding, editing, removing games which belong to event

### PR DESCRIPTION
**Pull request by maciek1995**
Creating relation between event and games
Adding games to event with possibility of searching games
Editing, removing games linked to event
Testing games controller
#24

### CODE REVIEW

**Comment by bpietraga** [comment]
+1

**Comment by hav3loc** [comment]
+1

File: app/policies/application_policy.rb [hunk 1]
```diff
@@ -6,48 +6,13 @@ def initialize(user, record)
     @record = record
   end
 
-  def index?
-    false
+  def game_host?
+    record.event.users.includes(user)
```
Comment (hav3loc) [inline app/policies/application_policy.rb]: Very clean way to do it, I'll steal it for my code!

**Comment by KamilFilip** [comment]
+1

**Pull request merged**